lumigator/frontend/nginx.conf

```diff
@@ -0,0 +1,45 @@
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri $uri/ /index.html;
+    }
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
+    #
+    #location ~ \.php$ {
```

Comment:
Great work @peteski22! Please delete these comments if they are not related to our solution

Reply:
Thanks Mario, I just took the entirety of the `default.conf` in case we ever wanted to diff it later in case things changed across nginx versions, but I can ditch the comments no problem! 😄